DSharpPlus/Entities/Guild/DiscordGuild.cs

```diff
@@ -924,6 +924,22 @@ public async Task<IReadOnlyList<DiscordRole>> ModifyRolePositionsAsync(IDictiona
         this._roles = new(returnedRoles.Select(x => new KeyValuePair<ulong, DiscordRole>(x.Id, x)));
         return returnedRoles;
     }
+    
+    /// <summary>
+    /// Removes a specified member from this guild.
+    /// </summary>
+    /// <param name="member">Member to remove.</param>
+    /// <param name="reason">Reason for audit logs.</param>
+    public async Task RemoveMemberAsync(DiscordMember member, string? reason = null)
```

Comment:
Maybe make this a `DiscordUser`

Reply:
the ID overload already covers that and more, so

Reply:
yeah but we only need the id anyways and `DiscordUser` would be the least specific type